inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/skip_tests_check.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gna/gna_config.hpp>
```

Comment:
This file already exists in the parent directory :)

Reply:
Right, leftover from using it before merging the other branch.